Mathlib/Algebra/Homology/Homotopy.lean

```diff
@@ -542,9 +542,8 @@ def mkInductiveAux₂ :
     ⟨(P.xNextIso rfl).hom ≫ I.1, I.2.1 ≫ (Q.xPrevIso rfl).inv, by simpa using I.2.2⟩
 #align homotopy.mk_inductive_aux₂ Homotopy.mkInductiveAux₂
 
--- Adaptation note:
--- During the port we marked these lemmas with `@[eqns]` to emulate the old Lean 3 behaviour.
--- See https://github.com/leanprover-community/mathlib4/issues/11647
+-- Porting note(#11647): during the port we marked these lemmas
```

Comment:
I think these should rather be porting notes. (Also applies in a few more places.)
I could be wrong about these, though!